src/Kunstmaan/GeneratorBundle/Command/KunstmaanGenerateCommand.php

```diff
@@ -852,11 +888,16 @@ protected function getEntityFields(
      */
     protected function getAvailableTemplates(BundleInterface $bundle)
     {
-        $configs = array();
+        $configs = [];
         $counter = 1;
 
         // Get the available sections from disc
-        $dir = $bundle->getPath().'/Resources/config/pagetemplates/';
+        if (Kernel::VERSION_ID >= 40000) {
+            $dir = $bundle->getPath().'/../config/kunstmaancms/pagetemplates/';
```

Comment:
```suggestion
                $dir = $this->container->getParameter('kernel.project_dir') . '/config/kunstmaancms/pagetemplates/';
```

To keep the code inline with the check in [`KunstmaanGenerateCommand`](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7b48b9c1d0bcecca7970011e8ef46d818d3a9ebb/src/Kunstmaan/GeneratorBundle/Command/KunstmaanGenerateCommand.php#L351)

Reply:
Comment above is still todo